src/components/Products/Products.component.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Pagination from "../Pagination/Pagination.component";
+// import products from "../../mocks/products.json";
+
+function Products(params) {
+  const { results } = params.data
+  const categories = params.categories || []
+
+  return (
+    <div className="featured">
+      <h2>{params.header}</h2>
+      <div className="featured-products">
+        {results
+          .filter((item) => {
```

Comment:
i'm not 100% sure but i think your `return true` its not really doing anything, you could try something like this
```suggestion
          .filter((item) => { categories.length > 0 && categories.includes(item.data.category.slug)
```
sorry if there is any typing error

Reply:
The return true is basically to show all the available products initially when there are **no** category selected, but I did made a shorter return:

`return (
              prodCategories.length === 0 ||
              prodCategories.includes(item.data.category.slug)
            );`